src/main/java/UserCommand.java

```diff
@@ -0,0 +1,3 @@
+public enum UserCommand {
+    TODO, DEADLINE, EVENT, LIST, MARK, UNMARK, DELETE, BYE
```

Comment:
Interesting use of enums. It seems you have attempted to perform the following

`command = UserCommand.valueOf(scanner.next().toUpperCase());`

without assigning values to the enum first :D

I myself am not clear if this is the right approach. However, I believe this is one way to solve it. 
```
public enum UserCommand {
    TODO("todo"),
    DEADLINE("deadline"),
    EVENT("event"),
    LIST("list"),
    MARK("mark"),
    UNMARK("unmark"),
    DELETE("delete"),
    BYE("bye");
    
    String value;
    
    UserCommand(String value) {
        this.value = value;
    }
}
```

Do let me know if you think otherwise!